services/core-api/app/api/now_applications/models/now_application_document_type.py

```diff
@@ -56,6 +56,8 @@ def transform_variables_to_data(now_application, draft_permit, mine, total_liabi
                 now_application.mine_name,
                 'mine_no':
                 now_application.mine_no,
+                'application_type':
+                now_application.notice_of_work_type.description,
                 'proposed_annual_maximum_tonnage':
```

Comment:
wait...why were the variables removed? They must exist here on the BE, and on the FE in the variable condition menu.